conda_smithy/lint_recipe.py

```diff
@@ -462,12 +462,12 @@ def lintify(meta, recipe_dir=None, conda_forge=False):
             filtered_host_reqs = [
                 req
                 for req in host_reqs
-                if req == str(language) or req.startswith(str(language))
+                if req == str(language) or req.startswith(str(language) + " ")
```

Comment:
Why not do something like `if req.partition(" ")[0] == str(language)`?

Reply:
Thanks. That is much better